Sources/Widgets/SVG/SVGCustomLandmarkRepresentation/index.js

```diff
@@ -0,0 +1,85 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkSVGRepresentation from 'vtk.js/Sources/Widgets/SVG/SVGRepresentation';
+
+const { createSvgElement } = vtkSVGRepresentation;
+
+// ----------------------------------------------------------------------------
+// vtkSVGCustomLandmarkRepresentation
+// ----------------------------------------------------------------------------
+
+function vtkSVGCustomLandmarkRepresentation(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkSVGCustomLandmarkRepresentation');
+
+  publicAPI.render = () => {
+    const list = publicAPI.getRepresentationStates();
+
+    const coords = [];
+    for (let i = 0; i < list.length; i++) {
+      coords.push(list[i].getOrigin());
+    }
```

Comment:
`list.forEach((representation) => {
  coords.push(reprentation.getOrigin());
});`

Reply:
`const coords = list.map((l)-> l.getOrigin());`

Reply:
For some reason only the solution 1 with the foreach works. The second does not seem to let other parts of the code reuse coords, as it is said to be const and read-only.